app/views/landing_page/blocks/_hero.html.erb

```diff
@@ -0,0 +1,25 @@
+<%
+  add_view_stylesheet("landing_page/hero")
+%>
+
+<div class="app-b-hero">
+  <%= picture_tag({ class: "app-b-hero__imagewrapper"}) do %>
```

Comment:
One of the things I've struggled with in the "scaffolding" PR is what to add into the system tests so that we can test that the various different blocks are rendering as expected. Perhaps a loop around the blocks in the landing page fixture that checks for specific bits of CSS?

Could something like that be added here?

Reply:
Something like this? 03126f6ff4da76a19dc7b0b9f8d8d144926901cc